pact/Pact/Core/IR/Eval/CoreBuiltin.hs

```diff
@@ -907,15 +936,18 @@ coreReadDecimal :: (CEKEval step b i m, MonadEval b i m) => NativeFunction step
 coreReadDecimal info b cont handler _env = \case
   [VString s] -> do
     viewEvalEnv eeMsgBody >>= \case
-      PObject envData ->
+      PObject envData -> do
+        chargeGasArgs info $ GObjOp $ ObjOpLookup s $ M.size envData
```

Comment:
Hmm... 

This one I'm not sure whether we need to charge for this.

Maybe we do, but realistically, people pay for env data size in the payload, so I think we can forego charging simply for the object access here. Flat gas should be fine.

Reply:
Resolved. I think this is okay.